src/cmd/support_archive/builder.go

```diff
@@ -116,6 +119,19 @@ func getLogOutput(tarballToStdout bool, logBuffer *bytes.Buffer) io.Writer {
 	}
 }
 
+func getAppNameLabel(ctx context.Context, pods clientgocorev1.PodInterface) string {
```

Comment:
would it make sense to move this function to a common location, in case we need it later somewhere else?